contracts/RepaymentRouter.sol

```diff
@@ -109,12 +109,15 @@ contract RepaymentRouter is Pausable {
         }
 
         // Transfer amount to creditor
-        require(tokenTransferProxy.transferFrom(
+        if (!tokenTransferProxy.transferFrom(
             tokenAddress,
             msg.sender,
             beneficiary,
             amount
-        ));
+        )) {
+            LogError(uint8(Errors.PAYER_OWNERSHIP_OR_ROUTER_APPROVAL_MISSING), agreementId);
```

Comment:
I actually think this name is an no longer relevant vestige of when the repayment router had NFT payments natively enabled.  That's no longer the case for v1.  Please remove this check, as well as the error from the enum.

Reply:
Just to confirm, we'll want to throw if `transferFrom` fails. So `require` it to succeed but there's no need to log a specific error.